content/docs/dvclive/index.md

```diff
@@ -154,3 +154,16 @@ with Live(save_dvc_exp=True) as live:
 After you run your training code, all the logged data will be stored in the
 `dvclive` directory. Check the [DVCLive outputs](/doc/dvclive/how-it-works) page
 for more details.
+
+## Run with DVC
+
+You can [run experiments with DVC] using [pipelines]. See examples of how to
```

Comment:
It would be nice for us to have an explanation (or inside a link) on why it is needed